plugins/thunar-apr/thunar-apr-image-page.c

```diff
@@ -173,7 +172,9 @@ thunar_apr_image_page_init (ThunarAprImagePage *image_page)
   /* add labels for the Exif info */
   for (n = 0; n < G_N_ELEMENTS (TAIP_EXIF); ++n)
     {
-      label = gtk_label_new (_(TAIP_EXIF[n].name));
+      const gchar *exif_title = exif_tag_get_title ( TAIP_EXIF[n] );
```

Comment:
Well .. if you do not accept this changes, I'll prepare new pull request which only add comments fields to the dialog. So ?